File: Sakila_Net/Sakila_Net/VB_SakilaLib/Neg/NegAddress.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegAddress
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOAddress.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( address as ClAddress) as Boolean
                        Return DAO.DAOAddress.SelectOne(address)     
                    End Function
                    Public  Shared Function crear( address as ClAddress) as Integer
                        Return DAO.DAOAddress.SqlInsert(address)     
                    End Function
                    Public  Shared Function eliminar( address as ClAddress) as Integer
                        Return DAO.DAOAddress.SqlDelete(address)     
                    End Function
                    Public  Shared Function actualizar( address as ClAddress) as Integer
                        Return DAO.DAOAddress.SqlUpdate(address)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegAddress.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegAddress
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOAddress.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClAddress address) 
        {
            return DAO.DAOAddress.sqlSelect(address);
        }
        public  static int crear(ClAddress address)
        {
            return DAO.DAOAddress.sqlInsert(address);
        }
        public  static int eliminar(ClAddress address)
        {
            return DAO.DAOAddress.sqlDelete(address);
        }
        public  static int actualizar(ClAddress address)
        {
            return DAO.DAOAddress.sqlUpdate(address);
        }
    }
}
